.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
I'd say this one should be in `~/.gitignore_global`, see https://github.com/FerretDB/FerretDB/blob/main/.gitignore#L1 ;)

Reply:
I'll remove that before review. I wanted to move this file between workstations and I'm too lazy to use a pendrive :)